app/kudos/templates/kudos_details.html

```diff
@@ -7,8 +7,8 @@
     <link rel="stylesheet" href="{% static "v2/css/kudos/styles.css" %}" />
     <script src="{% static 'v2/js/tweenlite.js' %}"></script>
     <link rel="stylesheet" href="{% static "v2/css/jquery-ui.css" %}" />
-    <meta property="twitter:image" content="{% static kudos.image %}" />
-    <meta property="og:image" content="{% static kudos.image %}" />
+    <meta property="twitter:image" content="{{ kudos.img_url }}" />
```

Comment:
This metatags are already in cards_pic maybe we should pass the correct param there with the url.
Also not sure if should be using static to serve the images from the cdn?

Reply:
^^^ we absolutely should be serving this from the CDN xD